index.html

```diff
@@ -54,13 +58,13 @@ <h1>I’m Alexandra Glad to see you!</h1>
        </div>
```

Comment:
- [ ] Kindly make sure your button, has an an`<a>` tag or a `<button>` tag and it follows the design given.